locals.tf

```diff
@@ -0,0 +1,21 @@
+locals {
+  unauthorized_api_calls_base_exclusions = [
+    "($.sourceIPAddress != \"delivery.logs.amazonaws.com\")",
+    "($.eventName != \"HeadBucket\")"
+  ]
+
+  unauthorized_api_calls_extra_exclusions = [
+    for svc in var.unauthorized_api_calls_extra_excluded_services :
+    "($.eventSource != \"${svc}\")"
+  ]
+
+  unauthorized_api_calls_all_exclusions = join(
```

Comment:
how confident are we that this can't be injected to have unexpected behaviour (the consequence of malicious activity)?